eva-release/pom.xml

```diff
@@ -55,6 +55,10 @@
             <artifactId>springfox-swagger2</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>io.hypersistence</groupId>
+            <artifactId>hypersistence-utils-hibernate-52</artifactId>
+        </dependency>
```

Comment:
```suggestion
      
```
I guess this is not required to be added here as it has already been mentioned in the main pom.xml